packages/utils/src/formatters/formatDataForMutation.ts

```diff
@@ -138,9 +138,10 @@ const formatDataForMutation = (
         return result;
       }
 
-      if (Array.isArray(formatedFieldData) && R.equals(formatedFieldData, initialFieldData)) {
-        return result;
-      }
+      // TODO: need to discuss with core team how to handle case when filed `isList` and have `Default Value`
```

Comment:
Wouldn't that break current projects?

Reply:
the problem is that:

when there is a field in the `isList` table + there is a default value - then the server, when creating a record by default, added an array with one element

if comment that code -> then a mutation with an empty list will be sent -> this is how it works in production now